From: Caleb Connolly <caleb@connolly.tech>
Date: Thu, 15 Jul 2021 16:49:11 +0100
Subject: [PATCH 026/108] arm64: dts: qcom: sdm845-oneplus-fajita: remove
 panel reset gpio

Don't specify the reset GPIO for the OnePlus 6T, the panel in the 6T
will refuse to initialise if it has been reset so use this as a
workaround.

Signed-off-by: Caleb Connolly <caleb@connolly.tech>
---
 arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi   | 2 --
 arch/arm64/boot/dts/qcom/sdm845-oneplus-enchilada.dts | 2 ++
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
index d92d961..5e100a2 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
@@ -332,8 +332,6 @@ display_panel: panel@0 {
 
 		vddio-supply = <&vreg_l14a_1p88>;
 
-		reset-gpios = <&tlmm 6 GPIO_ACTIVE_LOW>;
-
 		pinctrl-names = "default";
 		pinctrl-0 = <&panel_reset_pins &panel_te_pin &panel_esd_pin>;
 
diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-enchilada.dts b/arch/arm64/boot/dts/qcom/sdm845-oneplus-enchilada.dts
index bf2cf92..776a2c1 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-enchilada.dts
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-enchilada.dts
@@ -27,6 +27,8 @@ &display_panel {
 	status = "okay";
 
 	compatible = "samsung,sofef00";
+
+	reset-gpios = <&tlmm 6 GPIO_ACTIVE_LOW>;
 };
 
 &bq27441_fg {
